common/config/webpack/sampleapp.webpack.config.js

```diff
@@ -18,7 +18,7 @@ const webpackConfig = (sampleAppDir, env, babelConfig) => {
       minimizer: []
     },
     mode: env.production ? 'production' : 'development',
-    ...(env.production || !env.development ? {} : { devtool: 'eval-source-map' }),
+    ...(env.production || !env.development ? { devtool: 'source-map'} : { devtool: 'eval-source-map' }),
```

Comment:
we need to create source-maps for production builds. We were previously only doing this for development.